getting-started/templates/sizing-examples/data-management-sizing-example.yaml

```
@@ -177,6 +177,19 @@ dataframeservice:
     optimizeQueue:
       ## @param dataframeservice.workloadManagement.optimizeQueue.concurrencyLimit Number of parallel jobs processed from the queue.
       concurrencyLimit: 15
+    # Dremio targets high-cost queries to this queue, such as decimated queries
+    # that scan tables with more than 6 million rows.
```

```suggestion
    # that scan tables with more than 1 million rows.
```

With the DFS3 testing, I was noticing that all of the queries to the tables with 1 million rows were getting bucketed as "high-cost". This is super rough guidance, since Dremio's query cost calculation is opaque to us and probably a function of many things, but I think a 1 million row cutoff makes more sense given what I saw with DFS3.